src/common/transformations/include/transformations/convert_precision.hpp

```diff
@@ -81,22 +81,27 @@ class ov::pass::ConvertPrecision : public ov::pass::ModelPass {
     ConvertPrecision(ov::element::Type_t from,
                      ov::element::Type_t to,
                      type_to_fuse_map additional_type_to_fuse_map = {},
-                     bool keep_precision_sensitive_in_fp32 = false)
+                     bool keep_precision_sensitive_in_fp32 = false,
+                     bool convert_input_output_precision = true)
```

Comment:
do we plan to delete `convert_input_output_precision` mode? or is it useful for some cases?